vendors/sensecap/drivers/s2120/examples.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "type": "uplink",
+    "description": "uplink containing only pulse counter",
+    "input": {
+      "bytes": [
+        48,
+        49,
+        48,
+        51,
+        49,
+        48,
+        53,
+        48,
+        52,
+        54,
+        48,
+        48,
+        48,
+        48,
+        48,
+        48,
+        48,
+        55,
+        48,
+        48,
+        54,
+        52,
+        48,
+        48,
+        48,
+        97,
+        48,
+        48,
+        56,
+        53,
+        54,
+        97
+      ],
+      "fPort": 1,
+      "recvTime": "2020-08-02T20:00:00.000+05:00"
+    },
+    "output": {
+      "data": {
+        "0": {
+          "measurementValue": 808464.688,
+          "measurementId": 3,
+          "type": "Measurement"
+        },
+        "1": {
+          "measurementValue": 808464.438,
+          "measurementId": 1,
+          "type": "Measurement"
+        }
+      },
+      "errors": [],
+      "warnings": []
+    }
+  }
+]
```

Comment:
The output in this example does not fit with the real one. Here is the test results: 

```
● Decode uplink › uplink containing only pulse counter

    expect(received).toStrictEqual(expected) // deep equality

    - Expected  -  6
    + Received  + 21

      Object {
        "data": Object {
          "0": Object {
    -       "measurementId": 3,
    -       "measurementValue": 808464.688,
    -       "type": "Measurement",
    +       "measurementId": "4097",
    +       "measurementValue": 78.4,
    +       "type": "Air Temperature",
          },
          "1": Object {
    -       "measurementId": 1,
    -       "measurementValue": 808464.438,
    -       "type": "Measurement",
    +       "measurementId": "4098",
    +       "measurementValue": 80,
    +       "type": "Air Humidity",
    +     },
    +     "2": Object {
    +       "measurementId": "4099",
    +       "measurementValue": 1174405120,
    +       "type": "Light Intensity",
    +     },
    +     "3": Object {
    +       "measurementId": "4190",
    +       "measurementValue": 0.7,
    +       "type": "UV Index",
    +     },
    +     "4": Object {
    +       "measurementId": "4105",
    +       "measurementValue": 10,
    +       "type": "Wind Speed",
          },
```

Reply:
I have noticed that the results are the opposite of the ones of S2100 device. Maybe you have reversed the `examples.json` file between the S2100 and S2120?